### ISSUE TITLE
cannot install: s6-svscan: warning: unable to iopause: Operation not permitted

### ISSUE THREAD

**Issue by slinky07** [alexjustesen/speedtest-tracker#339]
**Describe the bug**
cannot successfully install

**To Reproduce**
use the following docker compose 

```
version: '3.3'
services:
▏ ▏ speedtest-tracker:
▏ ▏ ▏ ▏ container_name: speedtest-tracker
▏ ▏ ▏ ▏ ports:
▏ ▏ ▏ ▏ ▏ ▏ - '80:80'
▏ ▏ ▏ ▏ ▏ ▏ - '8443:443'
▏ ▏ ▏ ▏ environment:
▏ ▏ ▏ ▏ ▏ ▏ - PUID=1000
▏ ▏ ▏ ▏ ▏ ▏ - PGID=1000
▏ ▏ ▏ ▏ volumes:
▏ ▏ ▏ ▏ ▏ ▏ - './speedtest-tracker:/config'
▏ ▏ ▏ ▏ image: 'ghcr.io/alexjustesen/speedtest-tracker:latest'
▏ ▏ ▏ ▏ restart: unless-stopped
```
used 80 cause 8080 already binded 
**Expected behavior**
having app running. I don't understand what's up with s6-svscan error.

**Environment (please complete the following information):**
 - OS: raspberry pi 4b
-  Architecture: armv7l
 - Browser safari
 - Version latest

**Screenshots**

**Logs**
[_speedtest-tracker_logs-2.txt](https://github.com/alexjustesen/speedtest-tracker/files/10430074/_speedtest-tracker_logs-2.txt)

**Additional context**
is searched the error online and stuff but I don't understand what im doing wrong. I tried with different ports and it's not fixing it. thanks a lot for any assistance. 
I should note, on my Mac m1, I am able to successfully run it with same docker compose except that ports is on 8080 instead of 80.

**Comment by alexjustesen** [comment]
Well that's a new one... I'll look into it this week.

**Comment by alexdelprete** [comment]
Could it be a permission error due to the PID=1000 /PGID=1000 used in the docker-compose? Does 1000/1000 actually exist at OS level? Does it have the right permissions? I don't know...just speculating...:)

**Comment by slinky07** [comment]
I am on Raspberry Pi OS with the following docker engine version 
```
Client: Docker Engine - Community
 Version:           20.10.22
 API version:       1.41
 Go version:        go1.18.9
 Git commit:        3a2c30b
 Built:             Thu Dec 15 22:27:56 2022
 OS/Arch:           linux/arm
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.22
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.18.9
  Git commit:       42c8b31
  Built:            Thu Dec 15 22:25:44 2022
  OS/Arch:          linux/arm
  Experimental:     false
 containerd:
  Version:          1.6.15
  GitCommit:        5b842e528e99d4d4c1686467debf2bd4b88ecd86
 runc:
  Version:          1.1.4
  GitCommit:        v1.1.4-0-g5fd4c4d
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

```        #####           pi@raspberrypi
       #######          --------------
       ##O#O##          OS: Linux (Unknown) armv7l
       #######          Host: Raspberry Pi 4 Model B Rev 1.4
     ###########        Kernel: 5.15.68-v7l+
    #############       Uptime: 11 days, 3 hours, 18 mins
   ###############      Packages: 1944 (dpkg), 2 (snap)
   ################     Shell: bash 5.0.3
  #################     Resolution: 1920x1080
#####################   Terminal: /dev/pts/0
#####################   CPU: BCM2711 (4) @ 1.500GHz
  #################     Memory: 1262MiB / 7898MiB
```

**Comment by alexjustesen** [comment]
Try running `id usernamehere` so if your default user on your Pi is `root` you can do `id root` and it will tell you what your PDIG and PUID are.

**Comment by alexdelprete** [comment]
> am not certain if the PID=1000 /PGID=1000 comment is right tr not tbh

PID/PGID are used for this: 

![image](https://user-images.githubusercontent.com/7027842/213172981-1f6e8f7d-a3a2-45c6-97c8-4e499f7a1dd6.png)

([source](https://docs.linuxserver.io/general/understanding-puid-and-pgid))

**Comment by slinky07** [comment]
thanks, will try all that after work or in the morning!

**Comment by slinky07** [comment]
my uid and gid are as following 
when I run `id $user` I get : 
`uid=1000(pi) gid=1000(pi)`
seems like that should be fine? Im under ssh tunnel so might be a bit difficult to change my uid, from my understanding. 
let me know if that could be the user

**Comment by alexjustesen** [comment]
@slinky07 is docker running as the root user on your machine or as another?

**Comment by slinky07** [comment]
yeah it is running as root. I even did `sudo user mod -aG docker pi` but that didn't do anything. 

but I was able to fix it! was a headache but the problem is libseccomp2, on pi some people have issues with it. 

I tried to upgrade it to 2.5.4-1 as https://github.com/oznu/docker-homebridge/issues/434#issuecomment-1111958145 suggests, but that didn't work and I had depedency issues as I have the 2.3.3-4 version 
```
dpkg: dependency problems prevent configuration of libseccomp-dev:armhf:
 libseccomp-dev:armhf depends on libseccomp2 (= 2.5.4-1+b2); however:
  Version of libseccomp2:armhf on system is 2.3.3-4.
```

there's a few other things I tried from that thread, like https://github.com/oznu/docker-homebridge/issues/434#issuecomment-1112571949 which didn't work too 

only https://github.com/oznu/docker-homebridge/issues/434#issuecomment-1120134376 worked, which seems like a temporary solution/workaround, but it works. 
I guess I just gotta find how to manually upgrade libseccomp.
